### ISSUE TITLE
Add customizable brand text

### ISSUE THREAD

**Issue by rlalik** [ellmetha/django-machina#122]
I would propose to enclose brand name in following line

`<a class="navbar-brand" href="/">django-machina</a>`

of the `board_base.html` template in the `brand` block to allow easily overwrite it, e.g.

`<a class="navbar-brand" href="/">{% block brand %}django-machina{% endblock %}</a>`

**Comment by ellmetha** [comment]
Or something like:

```
<a class="navbar-brand" href="/">{% block brand %}{{ MACHINA_FORUM_NAME|default:"My forum" }}{% endblock %}</a>
```

Would you like submitting a PR?

**Comment by ellmetha** [comment]
Fixed in 72be033. Thanks for reporting! 😉
